openghg/standardise/footprint/_paris.py

```diff
@@ -0,0 +1,175 @@
+        inlet: Height above ground level in metres. Format 'NUMUNIT' e.g. "10m"
+        met_model: Underlying meteorlogical model used (e.g. UKV)
+        species: Species name. For a long-lived species this should be "inert".
+        network: Network name
+        period: Period of measurements. Only needed if this can not be inferred from the time coords
+        continuous: Whether time stamps have to be continuous.
+        high_spatial_resolution : Indicate footprints include both a low and high spatial resolution.
+        time_resolved: Indicate footprints are high time resolution (include H_back dimension)
+            Note this will be set to True automatically if species="co2" (Carbon Dioxide).
+        high_time_resolution:  This argument is deprecated and will be replaced in future versions with time_resolved.
+        short_lifetime: Indicate footprint is for a short-lived species. Needs species input.
+            Note this will be set to True if species has an associated lifetime.
+    Returns:
+        dict: Dictionary of data
+    """
+
+    if high_time_resolution:
+        warnings.warn(
+            "This argument is deprecated and will be replaced in future versions with time_resolved.",
+            DeprecationWarning,
+        )
+        time_resolved = high_time_resolution
+
+    xr_open_fn, filepath = check_function_open_nc(filepath)
+
+    with xr_open_fn(filepath) as temp:
+        fp_data = temp
+
+    time_resolved = check_species_time_resolved(species, time_resolved)
+    short_lifetime = check_species_lifetime(species, short_lifetime)
+
+    dv_rename = {"srr": "fp"}
+
+    attribute_rename = {"fp_output_units": "LPDM_native_output_units"}
+
+    dim_rename = {"latitude": "lat", "longitude": "lon"}
+
+    dim_reorder = ("time", "height", "lat", "lon")
+
+    # Ignore type - dv_rename type should be fine as a dict but mypy unhappy.
+    fp_data = fp_data.rename(**dv_rename)  # type: ignore
+    fp_data = fp_data.rename(**dim_rename)
+
+    fp_data = fp_data.transpose(*dim_reorder, ...)
+
+    for attr, new_attr in attribute_rename.items():
+        if attr in fp_data:
+            fp_data.attrs[new_attr] = fp_data.attrs.pop(attr)
+
+    # Need to read the metadata from the footprints and then store it
+    # Do we need to chunk the footprints / will a Datasource store it correctly?
+    metadata: Dict[str, Union[str, float, List[float]]] = {}
+
+    metadata["data_type"] = "footprints"
+    metadata["site"] = site
+    metadata["domain"] = domain
+    metadata["model"] = model
+
+    # Include both inlet and height keywords for backwards compatability
+    metadata["inlet"] = inlet
+    metadata["height"] = inlet
+    metadata["species"] = species
+
+    if met_model is not None:
+        metadata["met_model"] = met_model
+
+    if network is not None:
+        metadata["network"] = network
+
+    # Check if time has 0-dimensions and, if so, expand this so time is 1D
```

Comment:
If a footprint doesn't have a time coordinate, should we raise a value error?

Reply:
Added an else to current if statement for this to raise a `ParseError`